File: 프로그래머스/lv1/42748. K번째수/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for(int i = 0; i < commands.length; i++) {
            int[] arr = new int[commands[i][1] - commands[i][0] + 1];
            int idx = 0;
            for(int j = commands[i][0] - 1; j < commands[i][1]; j++) {
                arr[idx++] = array[j];
            }
            Arrays.sort(arr);
            answer[i] = arr[commands[i][2] - 1];
        }
        return answer;
    }
}

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = new Array(commands.length);
    for(let i = 0; i < commands.length; i++) {
        answer[i] = array.slice(commands[i][0] - 1, commands[i][1])
                         .sort((a, b) => a - b)[commands[i][2] - 1];
    }
    return answer;
}